### ISSUE TITLE
Toast missing upon resetting user password

### ISSUE THREAD

**Issue by 0815pascal** [wepublish/wepublish#75]
**Expected behaviour:** Upon resetting password, a toast should appear indicating that the password has successfully been changed

Actual behaviour:  nothing happens

![185793db-72e1-4aae-b12e-fb6f2915fe57](https://user-images.githubusercontent.com/11842633/99074203-9ee54300-25b7-11eb-88c5-bae81b3fcc03.gif)

**Comment by watzak** [comment]
Already fixed I would say :)

**Comment by roosnic1** [comment]
Yes you're right :). Thanks @watzak. Would you like to contribute to the project?
